ios/GigyaSdkWrapper.swift

```diff
@@ -30,6 +32,12 @@ enum GigyaInterruptionsSupported: String {
     case unknown
 }
 
+struct SessionModel: Codable {
```

Comment:
A public model **GigyaSession** is already present within the Swift SDK. No need to create a new one.

Reply:
Removed.